data/categories.txt

```diff
@@ -3207,7 +3207,7 @@ fi:Puhelin
 sw:Simu
 
 amenity-toilets
-en:3Toilet|wc|U+1F6BD|U+1F6BE|U+1F4A9|U+1F6BB|U+1F6B9|U+1F6BA
+en:3Toilet|wc|4restroom|4bathroom|U+1F6BD|U+1F6BE|U+1F4A9|U+1F6BB|U+1F6B9|U+1F6BA
```

Comment:
Я за 9 месяцев жизни в Англии ни разу не слышал loo. Но для нас важен поиск, то, что люди могут набрать.

> On 16.8.2016, at 08.56, therearesomewhocallmetim <notifications@github.com> wrote:
> 
> In data/categories.txt:
> 
> > @@ -3207,7 +3207,7 @@ fi:Puhelin
> >  sw:Simu
> >  
> >  amenity-toilets
> > -en:3Toilet|wc|U+1F6BD|U+1F6BE|U+1F4A9|U+1F6BB|U+1F6B9|U+1F6BA
> > +en:3Toilet|wc|4restroom|4bathroom|U+1F6BD|U+1F6BE|U+1F4A9|U+1F6BB|U+1F6B9|U+1F6BA
> Lavatory — да, довольно формальное слово. Насчет того, что loo - жаргон, не соглашусь. Это разговорное слово. Жаргон — это язык узких социальных слоев (например, уголовный или профессиональный жаргон). Loo можно услышать на каждом шагу. Но на то они и синонимы :)
> 
> Но я не настаиваю, а предлагаю.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>